### ISSUE TITLE
Option to build the docker containers before starting

### ISSUE THREAD

**Issue by jtanios** [testcontainers/testcontainers-python#76]
When using this package to test a custom container, it'd be good to build the containers before starting the service every time the test runs to make sure any new changes to the Dockerfiles are tested.

I think this can be a param to the `DockerCompose` constructor:
`def __init__(self, filepath, compose_file_name="docker-compose.yml", build=False):`

If `True`, it would add a `--build` option to the `docker-compose` command inside `start()`

**Comment by tillahoffmann** [comment]
That sounds like a good idea. Fancy sending a PR?

**Comment by jtanios** [comment]
> That sounds like a good idea. Fancy sending a PR?

Sure, I'll submit one pretty soon.

**Comment by tillahoffmann** [comment]
Closing for now as `wontfix` (see #83 for details), but thank you for the contribution!
